text/0202-events-and-logs-api.md

```diff
@@ -0,0 +1,172 @@
+* Logger is the class responsible for creating events using Log records.
+
+![Events and Logs API classes](img/0202-events-and-logs-api.png)
+
+### LoggerProvider
+
+Logger can be accessed with an LoggerProvider.
+
+In implementations of the API, the LoggerProvider is expected to be the stateful object that holds any configuration.
+
+Normally, the LoggerProvider is expected to be accessed from a central place. Thus, the API SHOULD provide a way to set/register and access a global default LoggerProvider.
+
+Notwithstanding any global LoggerProvider, some applications may want to or have to use multiple LoggerProvider instances, e.g. to have different configuration (like LogRecordProcessors) for each (and consequently for the Loggers obtained from them), or because it's easier with dependency injection frameworks. Thus, implementations of LoggerProvider SHOULD allow creating an arbitrary number of instances.
+
+#### LoggerProvider operations
+
+The LoggerProvider MUST provide the following functions:
+
+* Get an Logger
+
+##### Get an Logger
+
+This API MUST accept the following parameters:
+
+- name (required): This name SHOULD uniquely identify the instrumentation scope, such as the instrumentation library (e.g. io.opentelemetry.contrib.mongodb), package, module or class name.  If an application or library has built-in OpenTelemetry instrumentation, both Instrumented library and Instrumentation library may refer to the same library. In that scenario, the name denotes a module name or component name within that library or application. In case an invalid name (null or empty string) is specified, a working Logger implementation MUST be returned as a fallback rather than returning null or throwing an exception, its name property SHOULD be set to an empty string, and a message reporting that the specified value is invalid SHOULD be logged. A library implementing the OpenTelemetry API may also ignore this name and return a default instance for all calls, if it does not support "named" functionality (e.g. an implementation which is not even observability-related). A LoggerProvider could also return a no-op Logger here if application owners configure the SDK to suppress telemetry produced by this library.
+
+- version (optional): Specifies the version of the instrumentation scope if the scope has a version (e.g. a library version). Example value: 1.0.0.
+- schema_url (optional): Specifies the Schema URL that should be recorded in the emitted telemetry
+- event_domain (optional): Specifies the domain for the events created, which should be added in the attribute `event.domain` in the instrumentation scope.
+- include_trace_context (optional): Specifies whether the Trace Context should automatically be passed on to the events and logs created by the Logger. This SHOULD be false by default.
+
+It is unspecified whether or under which conditions the same or different Logger instances are returned from this function.
```

Comment:
Please make sure the wording here matches what we have in [Get Tracer section](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#get-a-tracer) (it has changed recently after attributes were added).